contracts/MWWRegistarMetis.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.4 <0.9.0;
+
+import "./DIAOracleV2.sol";
+import "./MWWRegistarBase.sol";
+import "hardhat/console.sol";
```

Comment:
Remove if not being used

Reply:
Hmm, yes I did not know consol.sol was being used or not.

Reply:
And DIAOracleV2 is being used since it imports the contract.